### ISSUE TITLE
Manage the project temporarily until core team is back.

### ISSUE THREAD

**Issue by yodalee** [google/mozc#448]
Dear developers and users:

As you known in [Contribution.md](https://github.com/yodalee/mozc/blob/master/CONTRIBUTING.md), the mozc team is suspending pull request right now. It is due to Google policy and the time of recover is unknonw by now.
Recently, there are some issues like #441 which breaks the building in gcc 8. Even though there are patches to fix them, they are unable to be merged into mainline. I am currently a fulltime software engineer. Since I was one of the victim(?) of the issue, I think I am able to manage a fork in my repository. I will keep manage the repository until core team open accepting contribution. 
My fork is at [yodalee/mozc](https://github.com/yodalee/mozc). So far I have copied all issue in google/mozc to it and re-enable the CI system including travis-CI and appvenyor. I planed to maintain a "dev" branch that branch from current master state, all the modification will add to that branch and if core team is back to life, I will open a pull request to merge those change back to mainline. I think in short time there will be no big change, just fix some emergency issue like 441 and other dictionary inproper or incorrect word.

In short:
If you are finding place to make contribution and don't want to, you can go to [yodalee/mozc](https://github.com/yodalee/mozc).

**Comment by meoberto** [comment]
Gentoo already incorporates #441 into their repo, just to note the fact that distros are being forced to circumvent upstream in order to get things to work.

**Comment by Archibalde** [comment]
Is this Github repository abandoned?
